### DISCUSSION TITLE
Filtering JSON field when using PB as framework?

### DISCUSSION THREAD

**Discussion by apikee** [pocketbase/pocketbase#2011] [Q&A] [answered]
Hello, is there a simple way to filter a JSON field in PB framework the same as can be done through JS API?
```
await pb.collection("demo").getList(1, 50, {
    filter: "myjsonfield.property = 'hello' && myjsonfield.otherprop >= 40"
})
```
I really like this way of handling json data but would love to be able to do this from go code.
Thank you

**Comment by ganigeorgiev** [comment, marked as answer]
You can use [`json_extract`](https://www.sqlite.org/json1.html#jex) as part of your db query.

For example:

```go
app.Dao().RecordQuery(yourcollection).
	AndWhere(dbx.NewExp("json_extract('myjsonfield', '$.property') = {:property}", dbx.Params{
		"property": "hello",
	})).
	All(&records)
```

**Reply by apikee** [reply]
Great, I wasn't aware this is possible (generally with sqlite). Thanks
